src/System.Management.Automation/engine/ProcessCodeMethods.cs

```diff
@@ -58,11 +58,8 @@ internal static int GetParentPid(Process process)
             Diagnostics.Assert(process != null, "Ensure process is not null before calling");
             PROCESS_BASIC_INFORMATION pbi;
             int size;
-#if CORECLR
             var res = NtQueryInformationProcess(process.SafeHandle.DangerousGetHandle(), 0, out pbi, Marshal.SizeOf<PROCESS_BASIC_INFORMATION>(), out size);
-#else
-            var res = NtQueryInformationProcess(process.Handle, 0, out pbi, Marshal.SizeOf<PROCESS_BASIC_INFORMATION>(), out size);
```

Comment:
It seems we should return in ClrFacade.cs `internal static SafeHandle GetSafeProcessHandle(Process process)` - we removed '#if CORE' there too.